pupa/tests/scrape/test_people_org_scrape.py

```diff
@@ -143,3 +142,12 @@ def test_committee_add_member_name():
     assert get_pseudo_id(c._related[0].person_id) == {'name': 'John Adams'}
     assert c._related[0].organization_id == c._id
     assert c._related[0].role == 'member'
+
+def test_person_add_membership_name():
+    p = Person('Leonardo DiCaprio')
+    p.add_membership('Academy of Motion Picture Arts and Sciences', role='winner', start_date='2016')
+    p._related[0].validate()
```

Comment:
Just kept it from the orginal add_membership test:
https://github.com/opencivicdata/pupa/blob/master/pupa/tests/scrape/test_people_org_scrape.py#L49-L59

Sorry, I can def remove. I'm embarrassed that I don't understand all the testing pieces yet, but that's probably obvious